src/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import Chart from 'chart.js';
+import {Chart} from 'chart.js';
```

Comment:
I don't think this is needed, because this is the UMD entry point. But probably works either way.

Reply:
I changes this because IDE complained about it )